carts/src/routes/index.ts

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+import { videoGames } from "../models/games";
+
+const routes = express.Router();
+
+routes.get("/", (req, res) => {
+  res.json("Welcome to the store!");
+});
+
+routes.get("/cart-items", (req, res) => {
+  res.json(videoGames);
+});
+
+routes.get("/cart-items/:id", (req, res) => {
+  const itemId = parseInt(req.params.id);
+  const item = videoGames.find((game) => game.id === itemId);
+  if (item) res.status(200).json(item);
+  res.json(videoGames);
```

Comment:
Be sure to wrap in try/catch blocks. This res.json returns the entire video game array, even if there is no matching id sent. It also returns 200 code when calling an incorrect id, when it should return 404.